mkt/ratings/views.py

```diff
@@ -100,14 +100,13 @@ def get_app(self, ident):
         except Webapp.DoesNotExist:
             raise Http404
         current_region = get_region()
-        if ((not app.is_public() or
-             not app.listed_in(region=current_region)) and
+        if (not app.is_public() and
                 not check_addon_ownership(self.request, app)):
             # App owners and admin can see the app even if it's not public
             # or not available in the current region. Regular users or
             # anonymous users can't.
             raise PermissionDenied(
-                'The app requested is not public or not available in region '
+                'The app requested is not public '
                 '"%s".' % current_region.slug)
```

Comment:
Change this to:
`raise PermissionDenied('The app requested is not public')`

(Removing the `'"%s".' % current_region.slug)` part)

Then you can make the change I suggest in test_views.py above.